src/dotless.Test/Specs/RulesetsFixture.cs

```diff
@@ -135,5 +135,135 @@ public void IESlash9HackAllowedForNonEvaluatedValues()
             var expected = input;
             AssertLess(input, expected);
         }
+
```

Comment:
Question. Currently its AppendMany in output that actually does the concatenation of values, but to be able to use separate chars for joining the values I would either have to move this up into the AppendCSS method, or make the output.AppendMany look at each item to be appended for the relevant join char?

Example

a {
font+: italic;
font+: 1em;
font+: Arial;
font+_: Verdana;
}

This should result in
font: italic 1em Arial, Verdana;

Notice the , between Arial and Verdana.

I am inclined to work with output to not break overall structure but do you see any problem with adding that kind of awareness to the output class method?